<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewReservationDialog</class>
 <widget class="QDialog" name="NewReservationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nowa rezerwacja</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="batchTab">
      <attribute name="title">
       <string>Turnus</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QGroupBox" name="dateGroupBox">
         <property name="title">
          <string>Termin</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="beginLabel">
            <property name="text">
             <string>Data przyjazdu:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="endLabel">
            <property name="text">
             <string>Data odjazdu:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCalendarWidget" name="beginCalendarWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCalendarWidget" name="endCalendarWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="mainGroupBox">
         <property name="title">
          <string>Zamawiający</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="5" column="0" colspan="2">
           <widget class="QLineEdit" name="disabilityLineEdit">
            <property name="placeholderText">
             <string>Niepełnosprawność</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLineEdit" name="emailLineEdit">
            <property name="placeholderText">
             <string>e-mail</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="countryLineEdit">
            <property name="placeholderText">
             <string>Kraj</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameLineEdit">
            <property name="placeholderText">
             <string>Imię</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="cityLineEdit">
            <property name="placeholderText">
             <string>Miasto</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="phoneLineEdit">
            <property name="placeholderText">
             <string>Telefon</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="surnameLineEdit">
            <property name="placeholderText">
             <string>Nazwisko</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="postalCodeLineEdit">
            <property name="placeholderText">
             <string>Kod pocztowy</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="numberLineEdit">
            <property name="placeholderText">
             <string>Numer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="streetLineEdit">
            <property name="placeholderText">
             <string>Ulica</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="summaryGroupBox">
         <property name="title">
          <string>Podsumowanie</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="daysLabel">
            <property name="text">
             <string>Ilość dni:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="daysValueLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="priceLabel">
            <property name="text">
             <string>Koszt:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="priceValueLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="participantGroupBox">
         <property name="title">
          <string>Dodatkowi uczestnicy</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLineEdit" name="participantSurnameLineEdit">
            <property name="placeholderText">
             <string>Nazwisko</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="participantNameLineEdit">
            <property name="placeholderText">
             <string>Imię</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QListWidget" name="participantsListWidget">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>70</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="addParticipantPushButton">
            <property name="text">
             <string>Dodaj</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>surnameLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>streetLineEdit</tabstop>
  <tabstop>numberLineEdit</tabstop>
  <tabstop>postalCodeLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>countryLineEdit</tabstop>
  <tabstop>phoneLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>disabilityLineEdit</tabstop>
  <tabstop>beginCalendarWidget</tabstop>
  <tabstop>endCalendarWidget</tabstop>
  <tabstop>participantSurnameLineEdit</tabstop>
  <tabstop>participantNameLineEdit</tabstop>
  <tabstop>addParticipantPushButton</tabstop>
  <tabstop>participantsListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
